src/components/pages/PageOrder/PageOrder.tsx

```diff
@@ -115,7 +115,7 @@ export default function PageOrder() {
       return;
     }
     const promises: any[] = [
-      axios.get(`${API_PATHS.product}/product`),
+      axios.get(`${API_PATHS.product}/products`),
```

Comment:
One of the points was "Your own Frontend application is integrated with product service (/products API) and products from product-service are represented on Frontend". Maybe I've misunderstood this one but I think that you should render what you get from backend.

Reply:
@j7amo I've added rendering of products based on data retrieved from back end.